### PR TITLE
Notebook kernel status, Create runtime network and API disable error message changes

### PR DESCRIPTION
**Pull request by Jeyaprakash-NK**
1. Added notebook kernel status on top of notebook.
2. Create runtime - Loader and error messages for networks and subnetworks issue.
3. Added notification messages on loading the plugin if the required APIs (Cloud Dataproc API, BigQuery API, Cloud Composer API and Dataproc Metastore API) are not enabled in the project.
4. Changed the default label name from dataproc-jupyter-plugin to bigquery-jupyter-plugin
5. Made changes to dynamically pick the version number from package.json file

### CODE REVIEW

File: dataproc_jupyter_plugin/controllers/composer.py [hunk 1]
```diff
@@ -33,10 +33,13 @@ async def get(self):
...
-                    env = environment.dict()
-                    response.append(env)
-                self.finish(json.dumps(response))
+                if type(environments) == list:
+                    for environment in environments:
+                        env = environment.dict()
+                        response.append(env)
+                    self.finish(json.dumps(response))
+                else:
+                    self.finish(environments)
```
Comment (ojarjur) [inline dataproc_jupyter_plugin/controllers/composer.py:42]: This is more complicated than it needs to be.

We can drop the iteration entirely and just set a default parameter to the `json.dumps` call:

```py
self.finish(json.dumps(environments, default=lambda x: x.dict()))
```

Comment (Jeyaprakash-NK) [reply dataproc_jupyter_plugin/controllers/composer.py]: Resolved

File: src/index.ts [hunk 2]
```diff
@@ -165,11 +169,116 @@ const extension: JupyterFrontEndPlugin<void> = {
...
+            credentials?.project_id
+          );
+      }
+
+      const composerListResponse =
+        await SchedulerService.listComposersAPICheckService();
+      const dataCatalogResponse =
+        await BigQueryService.getBigQuerySearchCatalogAPIService();
+
+      if (
```
Comment (VictorShiAmazingGrace) [inline src/index.ts:196]: These four categories of missing APIs seem awfully similar altogether. Do you mind drying them up a bit here for better readability? Thanks.

**Review by VictorShiAmazingGrace — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 2 above)
Comment (Jeyaprakash-NK) [reply src/index.ts]: Resolved

**Review by jinnthehuman — APPROVED** [review]

File: src/controls/NotebookButtonExtension.tsx [hunk 3]
```diff
@@ -125,6 +159,92 @@ class NotebookButtonExtensionPoint implements IDisposable {
...
+        ? ''
+        : '|';
+      kernelStatusValue.textContent =
+        this.context.sessionContext.kernelDisplayName.includes('Local')
+          ? ''
+          : status === 'idle'
+          ? status.charAt(0).toUpperCase() + status.slice(1) + ' (Ready)'
+          : status === 'busy'
+          ? status.charAt(0).toUpperCase() + status.slice(1) + ' (Executing)'
+          : status.charAt(0).toUpperCase() + status.slice(1);
```
[the remark this answers is not in the document]
Comment (Jeyaprakash-NK) [reply src/controls/NotebookButtonExtension.tsx]: Resolved

**Review by VictorShiAmazingGrace — APPROVED** [review]
LGTM

**Pull request merged**